src/discussions/comments/CommentsView.jsx

```diff
@@ -69,13 +69,18 @@ function DiscussionCommentsView({
   } = usePostComments(postId, endorsed);
   const sortedComments = useMemo(() => [...filterPosts(comments, 'endorsed'),
     ...filterPosts(comments, 'unendorsed')], [comments]);
+
+  const responsesText = () => (
+    <div className="mx-4 text-primary-700" role="heading" aria-level="2" style={{ lineHeight: '28px' }}>
+      {endorsed === EndorsementStatus.ENDORSED
+        ? intl.formatMessage(messages.endorsedResponseCount, { num: sortedComments.length })
+        : intl.formatMessage(messages.responseCount, { num: sortedComments.length })}
+    </div>
+  );
```

Comment:
nit: We do not need to make It function because we are not reusing it. so we can write like that 
{((hasMorePages && isLoading) || !isLoading) &&  (
      <div className="mx-4 text-primary-700" role="heading" aria-level="2" style={{ lineHeight: '28px' }}>
         {endorsed === EndorsementStatus.ENDORSED
          ? intl.formatMessage(messages.endorsedResponseCount, { num: sortedComments.length })
          : intl.formatMessage(messages.responseCount, { num: sortedComments.length })}
      </div>)}

Reply:
okay